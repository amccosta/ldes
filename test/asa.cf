x1=x1/100
x2=x2/100
L1=importance("oct1",x1,x2,2.8,0,0,-1,180)
IF (L1 > x0)
{
  PRINTVAR(x1)
  PRINTVAR(x2)
  PRINTVAR(L1)
  PRINT('')
}
IF (L1 <= x0)
{
  PRINTVAR(x1)
  PRINTVAR(x2)
  PRINT('+')
  PRINT('')
}
RETURN L1
