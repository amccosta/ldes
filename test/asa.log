9.02 1.44 2.03864e-05 
12.56 8.37 +
11.76 2.64 +
13.44 8.65 +
2.81 5.97 5.01809e-05 
3.54 1.37 +
4.11 2.85 +
10.66 2.37 +
8.82 1.41 +
10.92 6.37 +
2.08 7.91 +
8.96 7.35 +
2.97 7.39 +
2.63 7.38 +
8.42 4.23 +
11.39 6.62 +
8.63 8.1 +
10.75 2.74 +
8.92 2.9 +
11.32 2.49 +
12.36 5.44 +
5.59 7.77 5.68414e-05 
7.02 8.22 +
8.48 2.29 +
4.09 4.49 6.03545e-05 
3.99 1.06 +
8.12 4.94 +
4.18 5.07 6.31699e-05 
5.42 1.57 +
4.35 5.1 6.3899e-05 
4.6 5.92 6.54472e-05 
11.23 3.82 +
4.81 3.5 +
6.72 2.72 +
11.19 5.68 +
5.12 4.57 +
5.8 1.78 +
6.84 4.6 +
5.41 1.29 +
5.67 4.38 +
4.53 4.41 +
1.64 3.3 +
10.56 2.52 +
4.77 3.46 +
7.77 2.51 +
7.67 4.86 +
7.99 3.53 +
5.66 1.45 +
8.39 1.19 +
1.74 8.8 +
1.17 6.13 +
12.55 2.31 +
6.89 2.53 +
2.88 5.49 +
8.98 8.06 +
5.02 4.16 +
10.59 5.77 +
5.42 3.33 +
6.05 8.98 +
5.55 4.4 +
10.6 2.91 +
1.03 4.79 +
5.47 2.35 +
6.23 5.94 0.00295927 
7.08 6.77 +
5.71 5.71 0.0330648 
12.11 2.81 +
4.21 5.74 +
3.7 3.22 +
5.38 4.72 +
1.24 5.72 +
6.05 4.74 +
1.39 5.4 +
2.37 7.46 +
6.46 6.9 +
10.32 5.75 +
5.93 5.91 +
5.73 5.34 0.0340268 
7.7 4.36 +
5.66 5.11 +
6.71 5.05 +
4.03 5.29 +
5.42 6.43 +
5.3 5.86 +
5.77 5.36 +
2.08 5.34 +
6.25 5.23 +
5.79 5.3 +
5.7 8.76 +
5.92 4.66 +
4.9 5.24 +
7.05 6.46 +
4.75 3.66 +
5.55 5.43 0.0359541 
5.44 6.17 +
4.92 5.47 +
7.27 5.58 +
4.7 8.19 +
1.92 5.64 +
5.66 4.93 +
2.94 5.45 +
5.55 5.37 +
5.62 1.38 +
5.99 3.16 +
5.54 6.61 +
5.74 5.39 +
5.57 5.37 +
5.21 5.37 +
5.69 5.37 +
5.9 5.47 +
5.7 5.36 +
5.29 5.37 +
5.79 5.05 +
5.71 5.36 +
5.7 6.41 +
5.69 6.24 +
5.66 5.24 +
6 5.33 +
5.69 3.29 +
4.36 5.64 +
5.69 4.56 +
5.17 2.93 +
6.06 5.36 +
5.67 5.06 +
5.59 5.37 +
5.6 5.73 +
6.03 5.33 +
5.58 4.87 +
7.28 2.2 +
5.51 5.37 +
2.81 5.37 +
8.17 3.94 +
5.57 7.62 +
5.16 5.38 +
6.31 5.06 +
10.85 8.92 +
5.22 8.65 +
5.88 8.76 +
5.59 5.07 +
2.81 5.45 +
6.88 5.37 +
4.71 5.5 +
5.56 5.4 0.0359542 
4.68 5.24 +
5.59 5.5 0.0359544 
6.1 6.42 +
5.7 5.47 +
2.45 5.54 +
5.56 5.38 +
5.16 5.38 +
5.55 3.39 +
5.56 5.25 +
5.56 3.21 +
5.57 5.38 +
5.62 3.33 +
2.56 5.39 +
5.41 5.43 0.0359545 
5.4 5.15 +
3.95 5.44 +
12.64 5.15 +
5.46 5.43 +
4.69 5.39 +
5.41 5.41 +
6.18 5.34 +
5.55 5.41 +
5.51 5.36 +
5.55 5.4 +
5.53 5.4 +
6.19 6.4 +
5.52 4.62 +
2.59 5.67 +
5.91 5.4 +
5.45 5.74 +
5.53 5.4 +
5.52 5.4 +
5.52 5.38 +
6.37 4.18 +
5.52 5.38 +
8.03 5.38 +
5.49 5.38 +
6.91 5.39 +
3.82 5.38 +
5.52 5.37 +
5.52 5.38 +
5.53 5.79 +
6.18 5.25 +
5.59 5.38 +
5.57 5.38 +
5.5 5.38 +
5.57 5.38 +
7.29 5.38 +
6.93 5.39 +
6.98 5.43 +
5.57 5.38 +
4.35 3.32 +
5.57 5.54 +
5.57 5.52 0.0359549 
5.54 5.52 0.0359551 
5.54 5.52 +
5.54 5.53 +
5.59 5.54 +
7.43 5.24 +
5.47 5.52 +
5.56 5.36 +
5.54 5.52 +
5.43 5.28 +
6.73 5.49 +
5.54 6.27 +
5.54 6.78 +
5.56 5.52 +
5.54 5.47 +
8.05 5.53 +
7.4 5.56 +
5.54 5.52 +
5.21 5.57 +
5.54 5.62 +
4.54 5.57 +
5.54 5.52 +
8.7 5.15 +
5.49 5.52 +
5.43 5.73 +
5.46 5.53 +
5.19 5.23 +
5.54 5.52 +
5.54 5.52 +
5.54 5.52 +
5.22 5.54 +
5.54 5.52 +
5.54 5.52 +
5.54 5.52 +
5.54 5.09 +
5.53 5.54 +
5.54 5.58 +
5.05 4.43 +
5.54 5.52 +
4.59 5.52 +
5.54 4.65 +
5.99 5.51 +
5.54 5.52 +
1.23 6.54 +
5.53 5.52 0.0359552 
13.5 8.3 +
5.53 5.52 +
5.53 5.41 +
5.53 5.52 +
5.53 5.51 +
5.69 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.49 5.52 +
5.53 5.54 +
5.53 5.52 +
5.53 5.34 +
5.53 5.52 +
4.9 5.52 +
6.13 5.52 +
4.77 5.41 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.72 +
5.53 5.52 +
5.07 5.52 +
8.2 5.52 +
5.49 7.35 +
5.52 5.52 +
5.46 5.52 +
5.53 5.52 +
6.22 6.09 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.73 +
6.75 5.45 +
5.53 5.71 +
5.53 5.52 +
5.44 3.92 +
8.08 5.52 +
5.53 5.52 +
5.68 5.52 +
5.53 4.39 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 3.78 +
5.53 4.84 +
5.53 5.52 +
5.53 5.52 +
5.53 5.48 +
5.53 5.52 +
5.35 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.51 5.52 +
5.53 5.52 +
7.33 5.12 +
5.53 5.52 +
5.53 5.69 +
5.53 5.52 +
5.53 5.52 +
5.53 2.08 +
5.53 5.52 +
5.53 5.52 +
6.48 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.34 +
5.53 5.52 +
5.53 5.52 +
5.53 3.67 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 4.25 +
5.53 5.52 +
5.31 5.52 +
5.53 4.77 +
5.53 5.52 +
4.98 5.52 +
5.53 5.5 +
5.53 5.52 +
5.85 5.52 +
5.55 5.52 +
3.19 5.52 +
5.53 5.52 +
5.39 5.52 +
5.53 5.52 +
5.53 7.67 +
5.53 5.52 +
5.45 5.4 +
5.53 5.52 +
5.53 5.54 +
1.53 5.52 +
5.53 5.34 +
5.53 5.52 +
5.31 5.51 +
9.27 5.52 +
5.6 5.28 +
5.53 5.4 +
5.46 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 6.31 +
5.53 5.52 +
5.53 5.53 +
5.53 5.52 +
3.45 5.52 +
5.53 6.49 +
5.53 5.52 +
4.8 5.52 +
5.53 5.34 +
5.53 5.52 +
5.53 5.98 +
5.53 5.52 +
5.53 5.52 +
5.51 4.4 +
5.53 5.49 +
4.87 8.3 +
5.53 5.52 +
2.72 5.67 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.27 +
5.53 5.52 +
5.53 5.53 +
5.53 5.52 +
5.53 5.52 +
5.68 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
4.09 5.53 +
5.53 5.52 +
5.53 5.52 +
5.53 5.52 +
5.47 5.52 +
5.53 5.55 +
5.53 5.59 +
5.53 5.61 0.0359552 
5.53 5.61 +
5.08 5.65 +
5.53 6.05 +
5.53 5.62 +
5.53 5.61 +
5.53 5.61 +
5.53 5.76 +
5.53 5.61 +
5.24 5.61 +
8.4 5.61 +
7.33 5.75 +
5.53 5.61 +
5.53 5.61 +
11.02 5.61 +
5.53 5.63 +
5.53 5.61 +
5.53 4.77 +
5.53 5.62 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.59 +
5.54 5.61 +
5.53 4.66 +
5.53 5.61 +
5.39 5.61 +
1.38 5.61 +
5.53 6.94 +
6.85 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.74 +
6.18 5.3 +
5.53 5.6 +
1.74 5.61 +
5.53 5.57 +
5.53 5.61 +
5.53 5.93 +
5.53 5.67 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
2.37 5.61 +
5.73 5.61 +
5.53 5.61 +
7.09 5.61 +
5.53 4.02 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 6.63 +
5.53 5.61 +
5.53 5.62 +
5.53 5.61 +
5.5 5.61 +
9.03 5.61 +
5.53 5.65 +
5.53 5.61 +
5.52 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.6 +
6.3 5.61 +
5.53 5.61 +
5.53 7.93 +
5.19 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.55 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
8.93 5.61 +
13.07 5.59 +
5.53 5.61 +
5.55 5.61 +
6.03 5.61 +
5.53 5.61 +
5.53 5.61 +
5.52 5.76 +
5.53 5.61 +
5.53 5.95 +
5.42 5.72 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.94 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 4.83 +
5.53 5.61 +
5.53 5.59 +
5.53 5.61 +
7.78 5.61 +
5.53 5.61 +
5.53 5.61 +
7.93 5.61 +
5.54 5.61 +
6.64 5.61 +
5.53 5.79 +
5.53 5.62 +
5.53 5.61 +
6.41 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.55 4.99 +
5.53 5.61 +
5.53 5.47 +
5.53 5.61 +
5.54 5.61 +
5.53 5.61 +
5.53 5.34 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
4.97 5.61 +
2.87 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.63 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
6.76 5.94 +
5.53 5.61 +
5.53 5.61 +
5.53 4.94 +
6.16 5.61 +
5.53 5.61 +
5.53 5.61 +
5.44 5.51 +
5.53 5.36 +
5.54 5.61 +
4.72 5.61 +
5.53 5.61 +
5.53 5.43 +
5.53 5.61 +
5.53 5.61 +
5.94 5.59 +
5.53 5.53 +
5.18 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.54 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
6.01 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
11.9 5.61 +
5.53 5.62 +
5.53 5.61 +
5.53 5.58 +
5.53 5.61 +
5.53 5.74 +
5.52 5.61 +
5.53 5.61 +
5.53 8.03 +
5.53 5.61 +
5.53 5.62 +
5.53 5.61 +
5.53 5.61 +
5.53 5.57 +
5.53 5.61 +
5.53 5.59 +
5.58 5.61 +
5.53 5.61 +
5.53 5.61 +
5.56 5.61 +
5.53 5.61 +
5.67 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.61 +
5.54 5.61 +
5.53 5.61 +
5.53 5.63 +
1.25 5.61 +
5.53 5.61 +
5.53 5.61 +
5.53 5.99 +
4.85 5.61 +
5.53 5.61 +
