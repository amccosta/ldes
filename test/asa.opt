Limit_Acceptances                          5000 
Limit_Generated                            20000 
Limit_Invalid_Generated_States             1000 
Accepted_To_Generated_Ratio                0.0001 

Cost_Precision                             0.0001 
Maximum_Cost_Repeat                        5 
Number_Cost_Samples                        10 
Temperature_Ratio_Scale                    0.001 
Cost_Parameter_Scale_Ratio                 1 
Temperature_Anneal_Scale                   100 

Include_Integer_Parameters                 0 
User_Initial_Parameters                    0 
Sequential_Parameters                     -1 
Initial_Parameter_Temperature              1 

Acceptance_Frequency_Modulus               1000 
Generated_Frequency_Modulus                20000 
Reanneal_Cost                              1 
Reanneal_Parameters                        1 

Delta_X                                    0.001 
User_Tangents                              0 
Curvature_0                                0 

___Define_below_if_OPTIONS_FILE_DATA=TRUE

number_parameters=*parameter_dimension     2 

Param#:Minimum:Maximum:InitialValue:IntegerorReal
 0             100            1400           750            1 
 1             100            900            500            1 

___Define_below_if_QUENCH_COST_and_OPTIONS_FILE_DATA=TRUE

User_Quench_Cost_Scale[0]=1.0              1 

___Define_below_if_QUENCH_PARAMETERS_and_QUENCH_COST_and_OPTIONS_FILE_DATA=TRUE

Param#:User_Quench_Param_Scale
 0             2 
 1             2 
