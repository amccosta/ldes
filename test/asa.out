		ADAPTIVE SIMULATED ANNEALING

/* $Id: asa.c,v 17.19 1999/02/16 01:25:11 ingber Exp ingber $ */

OPTIONS_FILE = 1
OPTIONS_FILE_DATA = 1
RECUR_OPTIONS_FILE = 0
RECUR_OPTIONS_FILE_DATA = 0
COST_FILE = 1
ASA_LIB = 0
HAVE_ANSI = 1
IO_PROTOTYPES = 0
TIME_CALC = 1
TIME_STD = 0
TIME_GETRUSAGE = 1
INT_LONG = 1
INT_ALLOC = 0
SMALL_FLOAT =        1e-18
MIN_DOUBLE =        1e-18
MAX_DOUBLE =        1e+18
EPS_DOUBLE =        1e-18
CHECK_EXPONENT = 0
NO_PARAM_TEMP_TEST = 0
NO_COST_TEMP_TEST = 0
SELF_OPTIMIZE = 0
ASA_TEST = 0
ASA_TEST_POINT = 0
ASA_TEMPLATE = 0
MY_TEMPLATE = 1
ASA_TEMPLATE_LIB = 0
ASA_TEMPLATE_ASA_OUT_PID = 0
ASA_TEMPLATE_MULTIPLE = 0
ASA_TEMPLATE_SELFOPT = 0
ASA_TEMPLATE_SAMPLE = 0
ASA_TEMPLATE_QUEUE = 0
ASA_TEMPLATE_PARALLEL = 0
ASA_TEMPLATE_SAVE = 0
USER_INITIAL_COST_TEMP = 0
RATIO_TEMPERATURE_SCALES = 0
USER_INITIAL_PARAMETERS_TEMPS = 0
DELTA_PARAMETERS = 0
QUENCH_PARAMETERS = 1
QUENCH_COST = 1
QUENCH_PARAMETERS_SCALE = 1
QUENCH_COST_SCALE = 1
OPTIONAL_DATA = 0
OPTIONAL_DATA_INT = 0
USER_COST_SCHEDULE = 0
USER_ACCEPT_ASYMP_EXP = 0
USER_ACCEPTANCE_TEST = 0
USER_GENERATING_FUNCTION = 0
USER_REANNEAL_COST = 0
USER_REANNEAL_PARAMETERS = 0
MAXIMUM_REANNEAL_INDEX = 50000
REANNEAL_SCALE =           10
ASA_SAMPLE = 0
ASA_QUEUE = 0
ASA_RESOLUTION = 0
FITLOC = 0
FITLOC_ROUND = 1
FITLOC_PRINT = 1
MULTI_MIN = 0
MULTI_NUMBER = 2
ASA_PARALLEL = 0
ASA_SAVE = 0
SYSTEM_CALL = 1
FDLIBM_POW = 0
FDLIBM_LOG = 0
FDLIBM_EXP = 0

ASA_PRINT = 1
USER_OUT = user.out
ASA_OUT = asa.out
USER_ASA_OUT = 0
ASA_PRINT_INTERMED = 1
ASA_PRINT_MORE = 0
G_FIELD = 12
G_PRECISION = 7

OPTIONS->Limit_Acceptances = 5000
OPTIONS->Limit_Generated = 20000
OPTIONS->Limit_Invalid_Generated_States = 1000
OPTIONS->Accepted_To_Generated_Ratio =       0.0001

OPTIONS->Cost_Precision =       0.0001
OPTIONS->Maximum_Cost_Repeat = 5
OPTIONS->Number_Cost_Samples = 10
OPTIONS->Temperature_Ratio_Scale =        0.001
OPTIONS->Cost_Parameter_Scale_Ratio =            1
OPTIONS->Temperature_Anneal_Scale =          100
OPTIONS->Include_Integer_Parameters = 0
OPTIONS->User_Initial_Parameters = 0
OPTIONS->Sequential_Parameters = -1
OPTIONS->Initial_Parameter_Temperature =            1
OPTIONS->Acceptance_Frequency_Modulus = 1000
OPTIONS->Generated_Frequency_Modulus = 20000
OPTIONS->Reanneal_Cost = 1
OPTIONS->Reanneal_Parameters = 1

OPTIONS->Delta_X =        0.001
OPTIONS->User_Tangents = 0
OPTIONS->Curvature_0 = 0


start_asa:time: 0h 0m 0s; incr: 0h 0m 0s
Initial Random Seed = 38576

*number_parameters = 2

index_v parameter_minimum parameter_maximum parameter_value parameter_type 
 0        100          		 1400         	 750          1      
 1        100          		 900          	 500          1      


OPTIONS->User_Quench_Param_Scale[0] =            2
OPTIONS->User_Quench_Param_Scale[1] =            2

OPTIONS->User_Quench_Cost_Scale =            1

temperature_scale =    0.6907755
temperature_scale_parameters[0] =   0.06907755
*temperature_scale_cost =    0.6907755



:time: 0h 0m 0.012s; incr: 0h 0m 0.012s
*index_cost_acceptances = 0, *current_cost_temperature = 2.227492e-05
*accepted_to_generated_ratio =            1, *number_invalid... = 0
*number_generated = 0, *number_accepted = 0
best...->cost = -3.269843e-05, last...->cost = -3.269843e-05
index_v  best...->parameter current_parameter_temp	tangent
0	         208		           1	           0
1	         791		           1	           0


best...->cost=-3.458314e-05  *number_accepted=1  *number_generated=1
best...->cost=-4.541774e-05  *number_accepted=2  *number_generated=2
best...->cost=-5.684143e-05  *number_accepted=6  *number_generated=11
best...->cost=-6.035447e-05  *number_accepted=7  *number_generated=14
best...->cost=-6.316993e-05  *number_accepted=8  *number_generated=17
best...->cost=-6.389904e-05  *number_accepted=9  *number_generated=19
best...->cost=-6.544717e-05  *number_accepted=10  *number_generated=20
best...->cost=-0.002959271  *number_accepted=14  *number_generated=53
best...->cost=-0.03306483   *number_accepted=15  *number_generated=55
best...->cost=-0.03402678   *number_accepted=16  *number_generated=67
best...->cost=-0.03595408   *number_accepted=17  *number_generated=83
best...->cost=-0.03595416   *number_accepted=23  *number_generated=132
best...->cost=-0.03595439   *number_accepted=24  *number_generated=134
best...->cost=-0.03595454   *number_accepted=28  *number_generated=146
best...->cost=-0.0359549    *number_accepted=44  *number_generated=186
best...->cost=-0.03595513   *number_accepted=45  *number_generated=187
best...->cost=-0.03595516   *number_accepted=58  *number_generated=230
best...->cost=-0.03595517   *number_accepted=145  *number_generated=389

:time: 0h 0m 0.568035s; incr: 0h 0m 0.556035s
*index_cost_acceptances = 262, *current_cost_temperature = 3.103788e-10
*accepted_to_generated_ratio =    0.5566038, *number_invalid... = 0
*number_generated = 600, *number_accepted = 262
best...->cost =  -0.03595517, last...->cost =  -0.03595517
index_v  best...->parameter current_parameter_temp	tangent
0	         553		9.332543e-19	           0
1	         561		       1e-18	           0

curvature[0][0] =            0

curvature[1][0] =            0 	 = curvature[0][1]
curvature[1][1] =            0



 P_TEMP_TOO_SMALL exit_status = 1
current_user_parameter_temp[0] too small = 9.332543e-19
 Locate_Cost = 5, calculating curvatures while exiting asa ()
final_cost = best_generated_state->cost = -0.03595517 
*number_accepted at best_generated_state->cost = 145
*number_generated at best_generated_state->cost = 389
asa_end:time: 0h 0m 0.568035s; incr: 0h 0m 4.33681e-17s



